codegen.yml

```diff
@@ -21,8 +21,12 @@ generates:
       - typescript
       - typescript-graphql-request
       - typescript-operations
-  src/graphql/generated/masterchef.ts:
+  src/graphql/generated/master-chef.ts:
     schema: ${MASTERCHEF_SUBGRAPH_URL}
+    documents:
+      - src/graphql/fragments/master-chef.fragment.graphql
+      - src/graphql/fragments/pool.fragment.graphql
```

Comment:
i think these had been missing, not sure

Reply:
Yep, my bad on this. Thanks for adding